src/frontend/src/lib/components/wizards/auth/views/CreatePasskey.svelte

```diff
@@ -16,14 +18,19 @@
 
   let inputRef = $state<HTMLInputElement>();
   let name = $state("");
-  let creating = $state(false);
+  let isCreating = $state(false);
+  let isCancelled = $state(false);
 
   const handleCreate = async () => {
-    creating = true;
+    isCreating = true;
     try {
       await create(name.trim());
-    } finally {
-      creating = false;
+      isCreating = false;
+    } catch {
+      isCreating = false;
```

Comment:
I feel it's a bit assumption any error will be a cancelled error.

Are we sure we don't want this to manage all errors? Or maybe expect it as a prop "hasCancelled" and do the error management in the parent?